src/one_dragon/base/operation/one_dragon_context.py

```
@@ -297,11 +297,86 @@ def init(self) -> None:
+                        ["git", "config", "core.hooksPath", ".githooks"],
+                        capture_output=True,
+                        text=True,
+                        cwd=str(project_root)
+                    )
+                    if set_result.returncode == 0:
+                        log.info("Git hooks 路径已成功配置为 .githooks")
+                    else:
+                        log.warning(f"配置 Git hooks 失败 (返回码 {set_result.returncode}): {set_result.stderr.strip()}")
+                        return
+                elif current_hooks_path != ".githooks":
+                    # 检测到自定义 hooksPath，尝试自动将 pre-push 注入到对方的路径中以保证两者共存
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)